wemake_python_styleguide/visitors/ast/annotations.py

```diff
@@ -31,6 +31,11 @@ class _GenericAnnotationVisitor(BaseNodeVisitor):
         'typing_extensions.',
     ))
 
+    def visit_Assign(self, node: ast.Assign) -> None:
+        """Checks assignment patterns."""
+        self._check_annotation(node)
```

Comment:
Does it cover `x = y = Literal[None]` and `x, y = Literal[None], Literal[None]`?
We need to test it.

Reply:
It should. I'll add more tests.

Reply:
Added more tests.